packages/in_app_purchase/in_app_purchase_ios/ios/Classes/InAppPurchasePlugin.m

```diff
@@ -103,6 +111,14 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     [_paymentQueueHandler startObservingPaymentQueue];
   } else if ([@"-[SKPaymentQueue stopObservingTransactionQueue]" isEqualToString:call.method]) {
     [_paymentQueueHandler stopObservingPaymentQueue];
+  } else if ([@"-[SKPaymentQueue registerDelegate]" isEqualToString:call.method]) {
+    [self registerPaymentQueueDelegate];
+  } else if ([@"-[SKPaymentQueue removeDelegate]" isEqualToString:call.method]) {
+    [self removePaymentQueueDelegate];
+  } else if ([@"-[SKPaymentQueue showPriceConsentIfNeeded]" isEqualToString:call.method]) {
+    if (@available(iOS 13.4, *)) {
+      [_paymentQueueHandler showPriceConsentIfNeeded];
+    }
```

Comment:
Is it me or is the result callback missing.
`showPriceConsentIfNeeded` is a void so I would expect this:
```suggestion
    if (@available(iOS 13.4, *)) {
      [_paymentQueueHandler showPriceConsentIfNeeded];
    }
    result(nil);
```
or
```suggestion
    if (@available(iOS 13.4, *)) {
      [_paymentQueueHandler showPriceConsentIfNeeded];
      result(nil);
    }
    result(FlutterMethodNotImplemented);
```

Reply:
Fixed